public/actions/retriveKeyPair.ts

```diff
@@ -0,0 +1,16 @@
+import { KmsClient } from "cloudproof_js";
+import { BACKEND_URL } from "./backendConfig";
+import { KeyPair, KeysBytes } from "./types";
+
+//
+// Retrieve Key Pair
+//
+export const retrieveKeyPair = async (kmsToken: string, keysId: KeyPair): Promise<KeysBytes> => {
```

Comment:
Names are a bit confusing for me - why keysId has type KeyPair ? Are those keys or simple ids ?